kubemarine/core/flow.py

```diff
@@ -158,6 +161,14 @@ def create_empty_context(args: dict = None, procedure: str = None):
     }
 
 
+def get_task_list(tasks, _task_path=''):
```

Comment:
What about to show as tree?
```
tasks tree:
prepare
  check
    sudoer
    system
  ...
```

Reply:
discussed. decided to use list as it easy to use.